Select_Flick_Week2/ViewController.swift

```diff
@@ -195,8 +195,12 @@ extension ViewController : UITableViewDataSource {
         cell.leftSwipeSettings.transition = MGSwipeTransition.Rotate3D
         
         //configure right buttons
-        cell.rightButtons = [MGSwipeButton(title: "Share", icon: UIImageUtils.resizeImage(UIImage(named:"facebook")! , newWidth: 40) , backgroundColor: UIColor.blueColor(),  callback: {
+        cell.rightButtons = [MGSwipeButton(title: "Share", icon: UIImageUtils.resizeImage(UIImage(named:"share-white")! , newWidth: 40) , backgroundColor: UIColor.blueColor(),  callback: {
             (sender: MGSwipeTableCell!) -> Bool in
+            let sharingController = self.storyboard?.instantiateViewControllerWithIdentifier("ShareViewController") as! ShareViewController
+            self.navigationController?.pushViewController(sharingController, animated: true)
```

Comment:
uhm ... I don't use IB to add this behavior so I think it's still fine? or I misunderstood stm?